axelor-supplychain/src/main/resources/views/Selects.xml

```diff
@@ -66,7 +66,6 @@
 
 	<selection name="supplychain.sale.order.invoicing.wizard.select">
 		<option value="1">Invoice all</option>
-		<option value="2">Invoice a fraction</option>
```

Comment:
Update sequence of values and provide script  to update old values. 
Also check this value is used anywhere or not and remove it from static code from domain xml.